controllers/topics.controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectTopics } = require("../models/topic.model");
+
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then(({ rows: topics }) => {
```

Comment:
You could just change this to topics if you complete your model, you wouldn't need the {} either. So something like ```.then((topics)=>{res.status(200).send({topics})})```